anyway/models.py

```diff
@@ -1242,6 +1242,21 @@ def serialize(self):
         }
 
 
+class RoadJunctionKM(Base):
+    __tablename__ = "road_junction_km"
+    MAX_NAME_LEN = 100
+    road = Column(Integer(), primary_key=True, nullable=False)
```

Comment:
BigInteger?

Reply:
I agree, Integer is enough here, unless I missed anything :)